masking_package/background_functions.py

```diff
@@ -0,0 +1,149 @@
+'''Background functions, functions the user does not have to know about'''
+
+def reverse_to_default(user_input, supposed_level, default_value):
+	'''Tests whether the user input is of the desired value level (e.g., bool).
+	If it is not, the default value of the package is used, instead.'''
+	if isinstance(user_input, supposed_level):
+		return user_input
+	else:
+		print(f'Warning! The value "{user_input}" is not a(n) {supposed_level}.'
+			  + f'The program used the default value "{default_value}", instead.')
+		return default_value
+
+def draw_stim(image_stim_name, default_image_stim, location):
+	'''Displays image stimulus (mask, foil, target). If the entered object
+	is not an ImageStim (object type from PsychoPy), the example picture of
+	the package is used, instead.'''
+	if isinstance(image_stim_name, ImageStim):
+		p = image_stim_name
+		print(f'Warning, {image_stim_name} is not an object of class ImageStim!'
+		      + f'The program used the default picture "{default_image_stim}", instead.' )
+	else:
+		p = default_image_stim
+	#sets picture to the desired location
+	p.setPos(location)
+	p.draw()
```

Comment:
Maybe you can do this:
```suggestion
	return(p.draw())
```